packages/react/src/patterns/sub-patterns/ContentBlock/__stories__/ContentBlock.stories.js

```diff
@@ -45,13 +47,19 @@ storiesOf('Patterns (Sub-Patterns)|ContentBlock', module)
       none: null,
     };
 
+    const inverse = boolean('inverse', false);
+
     return (
-      <div className="bx--grid">
+      <div
+        className={cx('bx-grid', {
```

Comment:
Weighing in; Noticed the class name change in the snapshot, just wondered if such change was intentional. Otherwise:

```suggestion
        className={cx('bx--grid', {
```

Reply:
good catch, updated